questions.md

```diff
@@ -0,0 +1,124 @@
+<p align="center">
+<img src="logo.png">
+</p>
+
+
+babylon health
+==================================
+
+As described in the iOS recruitment process [main page](TODO), the face-to-face technical interviews involve us asking candidates a number of technical questions which we have decided to open-source.
+
+Please keep in mind that this document is in a continuous state of flux, with new questions being added, and old ones tweaked or removed, somewhat periodically.
```

Comment:
if it's "in flux", we can remove the `somewhat periodically `.

Reply:
Fixed in https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/2b7805ad040c2c104396fefee7a8f52b7286266c.